DevOps_Report.ps1

```diff
@@ -21,15 +21,15 @@ $orgURL = "https://dev.azure.com/{0}/" -f $OrganizationName
 
 $credentialName = $("git:https://{0}@dev.azure.com/{0}" -f $OrganizationName)
 
-$securePassword = Get-StoredCredential -Target $credentialName | Select -ExpandProperty password 
+$securePassword = Get-StoredCredential -Target $credentialName | Select-Object-ExpandProperty password 

```

Comment:
Needs a space between Select-Object and -ExpandProperty. Select is an alias and should be expanded though :)